src/main/java/StorageTest.java

```diff
@@ -0,0 +1,14 @@
+import org.junit.jupiter.api.Test;
```

Comment:
imports should be in this order:

import static org.junit.jupiter.api.Assertions.assertEquals;

import java.util.ArrayList;
import java.util.Arrays;

import org.junit.jupiter.api.Test;